Gemfile

```diff
@@ -2,14 +2,17 @@ source "https://rubygems.org"
 
 gemspec
 
+# for generating i18n files, gettext > 3.0 dropped ruby 1.8 support
```

Comment:
I think as of Katello 2.4, hammer has also been running inside of the ruby1.9.3 scl. Let me double check.